src/issue/screens/issue.screen.js

```diff
@@ -230,7 +230,7 @@ class Issue extends Component {
       navigation,
     } = this.props;
 
-    const isLoadingData = isPendingComments || isPendingIssue;
+    const isLoadingData = isPendingComments || isPendingIssue || false;
```

Comment:
Huh, weird... `!!` is probably the best solution for now, but I'm curious what the underlying issue is.